cli/src/pcluster/config/validators.py

```diff
@@ -714,7 +714,12 @@ def url_validator(param_key, param_value, pcluster_config):
 
     else:
         try:
-            urllib.request.urlopen(param_value)  # nosec nosemgrep
+            match = re.match(r"https://(.*?)\.s3\.(.*?)amazonaws\.com/(.*)", param_value)
```

Comment:
not supported by china partition: amazonaws.com.cn

Reply:
Done